src/extension.ts

```diff
@@ -107,3 +118,40 @@ function onConfigurationChange() {
         }
     });
 }
+
+/**
+ * Notifies user when Gauge VsCode Plugin updates.
+ *
+ * The functionality is to give users an option to see release notes when
+ * the version of the plugin changes irrespective of an upgrade or downgrade
+ *
+ * @param context ExtensionContext from activation
+ * @param latestVersion Latest Version of the plugin
+ * @return {void}
+ */
+function notifyOnNewGaugeVsCodeVersion(context: ExtensionContext, latestVersion: string) {
+    const gaugeVsCodePreviousVersion = context.globalState.get<string>(GAUGE_VSCODE_VERSION);
+
+    // First time installation
+    if (gaugeVsCodePreviousVersion === undefined) return;
```

Comment:
Should add the current gaugeVsCodeVersion to globalState for first time installation.